solutions/scripts/avn-dns-probe.sh

```diff
@@ -19,21 +22,22 @@ old_ip_num=$(echo ${old_ip} | wc -w | tr -d [:blank:])
 while [ 1 ]; do
     new_ip=$(dig +short $AVN_HOST | sort)
     new_ip_num=$(echo ${new_ip} | wc -w | tr -d [:blank:])
-
+    ./dd-custom-metric.sh aiven.node.count ${new_ip_num} > /dev/null
```

Comment:
What happens if the user doesn't want to send a DD custom metric?
Maybe we should add an argument to send it or not and then require the DD credentials?

Reply:
@Matthieu68857 thanks for catching this!  I initially wanted to make that as an example and optional that's why I put comment on top that to uncomment the dd script only if you want to, but I forgot to comment out dd script calls 😅